coda-prototype/model.ts

```diff
@@ -1810,6 +1810,135 @@ class StorageManager {
     }
 }
 
+class FileIO {
```

Comment:
How about calling it IOUtils, or FileUtils? FileIO feels a little redundant

Reply:
It was to distinguish it from StorageIO (if we renamed StorageManager), assuming the browser cache counts as IO. Otherwise I would be happy with something like FileUtils/StorageUtils